src/Rdd.Application/Controllers/AppController.cs

```
@@ -32,20 +32,34 @@ public AppController(IUnitOfWork unitOfWork, TCollection collection)
         public virtual async Task<TEntity> CreateAsync(ICandidate<TEntity, TKey> candidate, Query<TEntity> query)
         {
             var result = await Collection.CreateAsync(candidate, query);
+            if (result == null)
```

I prefer a throw variant, since DbContext can hold entities after a null return

---

@nfaugout what do you prefer ? null or throw ? I have to say, i wen null to be coherent with the rest and avoid unexpected exceptions on override, but override might face NulReferenceException instead, which is nothing but less explicit.

(also, the default POST path does not do any changes on the dbcontext, so i don't think this should be a big problem; event if people do use savechanges afterward, nothing should be commited)

---

I prefer throw rather than null, because null can be a valid returned value sometimes, whereas throwing an exception is explicit about the fact that the situation is not "normal".

---

allrighty